system-addon/test/unit/content-src/components/TopSites.test.js

```diff
@@ -0,0 +1,56 @@
+const React = require("react");
+const {shallow} = require("enzyme");
+const {_unconnected: TopSites, TopSite} = require("content-src/components/TopSites/TopSites");
+
+const DEFAULT_PROPS = {
+  TopSites: {rows: []},
+  dispatch() {}
+};
+
+describe("<TopSites>", () => {
+  it("should render a TopSites element", () => {
+    const el = shallow(<TopSites {...DEFAULT_PROPS} />);
+    assert.ok(el);
```

Comment:
Is just asserting that the wrapper exists a strong enough assertion?  Maybe assert on wrapper.name() or wrapper.hasClass("...")?